main.go

```diff
@@ -750,6 +584,23 @@ func StringPrompt(label string) string {
 }
 
 func main() {
+
+	viper.AddConfigPath(".")
```

Comment:
Add some comments in README.md to reflect what needs to be done add credentials to kelper.key, and key.yml files.